circle.yml

```diff
@@ -29,7 +29,7 @@ jobs:
             echo 'pip version: ' && python -m pip --version 
       - run:
           name: Installing cmake 
-          command: brew install cmake
+          command: brew install cmake || brew install cmake
```

Comment:
glad this works, but it's a weird command :smile: 
> do something, or do that same thing again ... ?

Reply:
It only retries the command if it doesn't work the first time.

Reply:
right..I'd assume computers be deterministic in this matter :P

Reply:
The brew error message told me to run it a second time.  It was a problem with updating cmake to latest version, afterwhich it didn't output some  critical file and failed.